sympy/calculus/singularities.py

```diff
@@ -32,3 +33,123 @@ def singularities(expr, sym):
                                   " implemented")
     else:
         return tuple(sorted(solveset(simplify(1/expr), sym)))
+
+###########################################################################
+###################### DIFFERENTIAL CALCULUS METHODS ######################
+###########################################################################
+
+
+def is_increasing(f, given_interval):
+    """
+    Returns if a function is increasing or not, in the given
+    `Interval`.
```

Comment:
Does it actually tell you both? If it only tells you the function is increasing, then the function name is fine, but if this tells you whether it is increasing or decreasing, then maybe you want a function like: `increasing_or_decreasing`. I don't think it is necessarily true that a function is decreasing if it is not increasing. It could be neither.